stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -200,6 +200,12 @@ internal class _AnyRandomAccessCollectionBox<Element>
 %   end
 
 %   override = 'override' if Kind != 'Sequence' else ''
+  internal ${override} func _drop(
+    while predicate: (Element) throws -> Bool
```

Comment:
All new predicates in this file should be `@noescape`.  Please also add dispatch tests in `validation-test/stdlib/ExistentialCollection.swift.gyb`.